### PR TITLE
Issue #5253: Enable telemetry module on new installs

### LINKED ISSUE #5253: Enable telemetry module on new installs (in 1.21.0)
This is part of the #5231 meta.

I think that we should tackle the following items here:
- [x] Enable telemetry module on new installs
- [x] Add easy opt-out to disable telemetry module
- [ ] Add Admin Dashboard block to telemetry module

### PR DESCRIPTION
**Pull request by docwilmot**
Fixes https://github.com/backdrop/backdrop-issues/issues/5253

### CODE REVIEW

**Comment by docwilmot** [comment]
Rebooted for PHP 5 fails.

File: core/modules/telemetry/block.telemetry.inc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   */
+  function getContent() {
+    $items = array();
+    if (user_access('access site reports')) {
+      $items[] = l(t('See what information is being sent to Backdrop CMS'), 'admin/reports/telemetry');
+    }
+    if (user_access('administer modules')) {
+      $items[] = l(t('Disable the telemetry module'), 'admin/modules');
+    }
+    $items[] = l(t('View the Backdrop CMS data collection policy'), 'https://backdropcms.org/philosophy');
```
Comment (quicksketch) [inline core/modules/telemetry/block.telemetry.inc]:
```suggestion
    $items[] = l(t('View the Backdrop CMS data collection policy'), 'https://backdropcms.org/privacy');
```

File: core/modules/telemetry/block.telemetry.inc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  /**
+   * {@inheritdoc}
+   */
+  function getContent() {
+    $items = array();
+    if (user_access('access site reports')) {
+      $items[] = l(t('See what information is being sent to Backdrop CMS'), 'admin/reports/telemetry');
+    }
+    if (user_access('administer modules')) {
+      $items[] = l(t('Disable the telemetry module'), 'admin/modules');
```
Comment (quicksketch) [inline core/modules/telemetry/block.telemetry.inc]: Module names are usually uppercase.

```suggestion
      $items[] = l(t('Disable the Telemetry module'), 'admin/modules');
```

**Comment by quicksketch** [comment]
Merged into 1.x for 1.21.0. Thanks @docwilmot and @klonos!

**Pull request merged**